### PR TITLE
Fix `force_aleternative` for experiments with incremented version

### PR DESCRIPTION
**Pull request by giraffate**
In dashboard, `force_alternative` doesn't work for experiments with incremented version to look for key without version.

This PR fixes this issue and `force_alternative` works for experiments with incremented version.

### CODE REVIEW

File: spec/dashboard_spec.rb [hunk 1]
```diff
@@ -74,17 +74,36 @@ def link(color)
...
+        allow(Split::User).to receive(:new).and_return(user)
+        experiment.increment_version
+      end
+
+      let!(:user) do
+        Split::User.new(@app, { "#{experiment.name}:#{experiment.version}" => 'a' })
+      end
+
+      it "should set current user's alternative" do
+        puts experiment.version
```
Comment (andrehjr) [inline spec/dashboard_spec.rb]: can you remove this debug code?

File: spec/dashboard_spec.rb [hunk 2]
```diff
@@ -74,17 +74,36 @@ def link(color)
...
-      expect(user[experiment.name]).to eq("b")
+    context "increment version" do
+      before do
+        allow(Split::User).to receive(:new).and_return(user)
+        experiment.increment_version
+      end
+
+      let!(:user) do
+        Split::User.new(@app, { "#{experiment.name}:#{experiment.version}" => 'a' })
+      end
```
Comment (andrehjr) [inline spec/dashboard_spec.rb]: Can you mode the `before block` after the `let!` line?

**Comment by giraffate** [comment]
Sorry, I find a mistake in this PR. Please wait for reviewing.

**Comment by giraffate** [comment]
This PR didn't take my own patch (https://github.com/splitrb/split/pull/567) into consideration. So I think that I will close this and resend another patch later.

**Pull request closed without merging**